primitives/src/channel_fixtures.rs

```diff
@@ -1,14 +1,16 @@
-//use chrono::{DateTime, Utc};
-//use fake::faker::*;
-//use time::Duration;
-//
-//use crate::asset::fixtures::get_asset;
-//use crate::fixtures::{get_targeting_tags, get_validator};
-//use crate::test_util;
-//use crate::BigNum;
-//
-//use super::{Channel, ChannelId, ChannelSpec, SpecValidators, ValidatorDesc};
-//
+use chrono::{DateTime, Utc};
+use fake::faker::*;
+use time::Duration;
+
+use crate::targeting_tag::fixtures::get_targeting_tags;
+use crate::validator::fixtures::get_validator;
+use crate::BigNum;
+
+use super::{Channel, ChannelSpec, SpecValidators, ValidatorDesc};
+use crate::util::tests::take_one;
+
+const ASSETS_LIST: [&str; 8] = ["DAI", "BGN", "EUR", "USD", "ADX", "BTC", "LIT", "ETH"];
```

Comment:
FYI the`depositAsset` would be an Eth address in production

Reply:
We do need some dummy data, so I am opened for suggestions to improve this.

Reply:
Yeah it's fine I was just pointing it out, because its not apparent at first glance

Reply:
It's not apparent with the `String` and no documentation comments on the `Channel` as well